dashboard/app/controllers/api/v1/assessments_controller.rb

```diff
@@ -187,4 +187,26 @@ def section_responses
   def section_surveys
     render json: LevelGroup.get_summarized_survey_results(@script, @section)
   end
+
+  def section_feedback
+    feedback = {}
+    @script.script_levels.each do |script_level|
```

Comment:
i think we want to extract this logic out of the controller and into a model. it looks like it would make the most sense in the `Script` model possibly. then we could have the controller call `@script.get_feedback_for_section` or something similar